commands/build_test.go

```diff
@@ -0,0 +1,31 @@
+package commands
```

Comment:
We need to validate that the label has been serialized correctly. I think this would be a reasonable place to check that.